README.md

```diff
@@ -144,24 +144,26 @@ This extension contributes the following settings that can be later updated in s
-| `idf.saveBeforeBuild`      | Save all the edited files before building (default `true`)                    |
-| `idf.coveredLightTheme`    | Background color for covered lines in light theme for gcov coverage           |
-| `idf.coveredDarkTheme`     | Background color for covered lines in dark theme for gcov coverage            |
-| `idf.partialLightTheme`    | Background color for partially covered lines in light theme for gcov coverage |
-| `idf.partialDarkTheme`     | Background color for partially covered lines in dark theme for gcov coverage  |
-| `idf.uncoveredLightTheme`  | Background color for uncovered lines in light theme for gcov coverage         |
-| `idf.uncoveredDarkTheme`   | Background color for uncovered lines in dark theme for gcov coverage          |
+| Setting ID                   | Description                                                                   |
+| ---------------------------- | ----------------------------------------------------------------------------- |
+| `idf.espIdfPath`             | Path to locate ESP-IDF framework (IDF_PATH)                                   |
+| `idf.toolsPath`              | Path to locate ESP-IDF Tools (IDF_TOOLS_PATH)                                 |
+| `idf.pythonBinPath`          | Python absolute binary path used to execute ESP-IDF Python Scripts            |
+| `idf.customExtraPaths`       | Paths to be appended to \$PATH                                                |
+| `idf.customExtraVars`        | Variables to be added to system environment variables                         |
+| `idf.useIDFKconfigStyle`     | Enable style validation for Kconfig files                                     |
+| `idf.showOnboardingOnInit`   | Show ESP-IDF Configuration window on extension activation                     |
+| `idf.adapterTargetName`      | ESP-IDF target Chip (Example: esp32)                                          |
+| `idf.openOcdConfigs`         | Configuration files for OpenOCD. Relative to OPENOCD_SCRIPTS folder           |
+| `idf.saveBeforeBuild`        | Save all the edited files before building (default `true`)                    |
+| `idf.coveredLightTheme`      | Background color for covered lines in light theme for gcov coverage           |
+| `idf.coveredDarkTheme`       | Background color for covered lines in dark theme for gcov coverage            |
+| `idf.partialLightTheme`      | Background color for partially covered lines in light theme for gcov coverage |
+| `idf.partialDarkTheme`       | Background color for partially covered lines in dark theme for gcov coverage  |
+| `idf.uncoveredLightTheme`    | Background color for uncovered lines in light theme for gcov coverage         |
+| `idf.uncoveredDarkTheme`     | Background color for uncovered lines in dark theme for gcov coverage          |
+| `idf.focusTaskOutput`        | Focus on Build/Flash task output on start                                     |
```

Comment:
lets remove `idf.focusTaskOutput ` settings because user, who wants to silent their notification can enable `idf.notificationSilentMode`

Reply:
But the purpose of the setting is different, focusTaskOutput is now only to show task output when build/flash happens.

Reply:
true but since `idf.notificationSilentMode ` does the same thing but on a global level, its not required to give settings to silent each and individual notifications (at least for now), we have have simple toggle, i.e., either users will get all notification or none (except for errors because they are important)